src/rippled/lib/utils.js

```diff
@@ -14,7 +14,11 @@ export const ACCOUNT_FLAGS = {
   0x00400000: 'lsfGlobalFreeze',
   0x00800000: 'lsfDefaultRipple',
   0x01000000: 'lsfDepositAuth',
-  0x02000000: 'lsfAMM',
+  0x04000000: 'lsfDisallowIncomingNFTOffer',
+  0x08000000: 'lsfDisallowIncomingCheck',
+  0x10000000: 'lsfDisallowIncomingNFTOffer',
+  0x20000000: 'lsfDisallowIncomingNFTOffer',
```

Comment:
Is this supposed to be `lsfDisallowIncomingNFTOffer` 3 times?

Reply:
Woops.